client/src/components/jobCard/jobCard.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+  Button,
+} from '@mui/material';
+import { Link } from 'react-router-dom';
+import './jobCard.css';
+
+function JobCard() {
+  return (
+    <div className="content">
+      <div className="job-card">
+        <Link to="/job/jobid">
+          <h2 className="h2">
+            Graphic Design
+          </h2>
+          <p className="description">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Impedit,
+            nisi molestiae praesentium eum repellat expedita sit reicien quas
+            perferendis beatae nam ipsum in neque cumque corporis id commod
+          </p>
+        </Link>
+        <div className="second-section">
+          <div className="budget-proposal-section">
+            <div className="proposals">
+              proposals:
+              <span>3</span>
+            </div>
+            <div className="budget">
+              budget:
+              <span>$50</span>
+            </div>
+          </div>
+          <Link to="/job/jobid">
```

Comment:
why not to add `onClick` button handler and then navigate to the desired route
check [useNavigate API](https://reactrouter.com/en/main/hooks/use-navigate)